style/style.css

```diff
@@ -1,3 +1,77 @@
+body {
+  /* border: 5px solid purple; */
+  height: 107vh;
+  background: #1c1a19;
+  background-image: url(../assets/Illustration.png);
+  background-position: right bottom;
+  background-repeat: no-repeat;
+  color: white;
+  margin: 0;
+}
+
+header {
+  background-color: #3c3a39;
+
+  /* border: 5px solid red; */
+  width: 100%;
+}
+
+nav {
+  background-color: #3c3a39;
+  width: 100%;
+  display: flex;
+  align-items: stretch;
+}
+
+nav > ul {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+
+  /* border: 5px solid red; */
+  list-style-type: none;
+  flex-direction: row;
+}
+
+nav > ul > li {
+  /* border: 5px solid blue; */
+  width: 77px;
+  height: 24px;
+}
+
+.my-logo {
+  font-family: Roboto, sans-serif;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0;
+}
+
 h1 {
-    color: blue;
-}
\ No newline at end of file
+  height: 124px;
+  font-family: Crete Round, sans-serif;
+  font-size: 56px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 64px;
+  letter-spacing: 0.3700000047683716px;
+  text-align: left;
+  color: #ff6b00;
+}
+
+section {
+  display: flex;
+
+  /* border: 5px solid red; */
```

Comment:
- [ ] For better code readability always make sure to remove the commented code. Kindly remove all the commented code.